src/btree/bt_prefetch.c

```diff
@@ -59,8 +52,10 @@ __wt_btree_prefetch(WT_SESSION_IMPL *session, WT_REF *ref)
             ret = __wt_conn_prefetch_queue_push(session, next_ref);
             if (ret == 0)
                 ++block_preload;
-            else if (ret != EBUSY)
-                WT_RET(ret);
+            else if (ret != EBUSY) {
+                WT_STAT_CONN_INCR(session, block_prefetch_page_not_queued);
+                continue;
```

Comment:
Why did we add a continue here? I don't think `ret` will be returned anymore if it's non-zero and also not `EBUSY`.

Reply:
Good point! [c101518](https://github.com/wiredtiger/wiredtiger/pull/9949/commits/c101518d8256eeacb3a2f77f9de61da5ef649f6c)